src/components/document/tile-row.tsx

```diff
@@ -121,6 +122,10 @@ export class TileRowComponent extends BaseComponent<IProps, IState> {
     this.setState({ tileAcceptDrop: tileId });
   }
 
+  private handleRequestRowHeight = (height: number) => {
```

Comment:
I think this callback should take a `tileId` as well. Ultimately there will need to be a negotiation over requests from multiple tiles in the same row. The image tile is more resizable than others, so if another tile in the same row wants to be a certain size, that should be taken into account.

Reply:
Makes sense. Yes, figured there would be a negotiation, but hadn't considered that we might want to know who was requesting the height.